packages/stencil/components/button/src/components/osds-button/osds-button.spec.ts

```diff
@@ -215,5 +219,21 @@ describe('spec:osds-button', () => {
       expect(controller.validateAttributes).toHaveBeenCalledWith();
       expect(controller.validateAttributes).toHaveBeenCalledTimes(1);
     });
+
+    it('should call handleClick of controller', async () => {
```

Comment:
here you are test the code of the two method, but you are not testing that a listen is set for keyup and click.

you can do this 2 more test with a test with something like `page.root.click()`.
for keyboard test :

```
        // Récupération de l'élément host du composant
        const host = page.root?.host;

        // Création et dispatch de l'événement KeyboardEvent
        const event = new KeyboardEvent('keydown', {
          key: 'Enter',
        });
        host.dispatchEvent(event);
await page.waitForChanges();
```

Reply:
Added these tests too